website/docs/community/people.md

```diff
@@ -19,4 +19,17 @@ I love spending time with students, teaching and finding ways to improve the cur
 
 I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.
 
-Apart from computer science, I really enjoy skiing, dancing and cooking.
\ No newline at end of file
+Apart from computer science, I really enjoy skiing, dancing and cooking.
```

Comment:
I don't know what happened here.

Reply:
copycat

Reply:
What is this?

Reply:
My guess is Liza's sentence finished with an EOF and I addedd a newline.